index.html

```diff
@@ -13,13 +13,2689 @@
+                  at Bar Brawl</a
+                >
+              </h5>
+            </div>
+          </div>
+          <div class="col-lg-4 col-md-4">
+            <div class="card my-1">
+              <a href="#">
+                <img
+                  class="card-img-top position-relative img-fluid"
+                  src="https://d.newsweek.com/en/full/1604708/down-jackson.webp?w=480&h=320&l=63&t=51&q=90&f=6aa994b38cbdb4a0635bc48065702d5a"
+                  alt="picture"
+                />
+              </a>
+            </div>
+
+            <div class="card-body p-0 m-0">
+              <h5 class="card-title">
+                <a href="#" class="font-weight-bold text-body"
+                  >Texas Republicans Rebuke Governor for Mask Mandate as
+                  COVID-19 Cases Spike</a
+                >
+              </h5>
+            </div>
+          </div>
+          <div class="col-lg-4 col-md-4 d-flex flex-column">
+            <div class="border-bottom pb-2">
+              <img
+                class="pt-2 mr-3 float-left img-fluid"
+                src="https://d.newsweek.com/en/full/1598987/george-washington.webp?w=95&h=95&f=6250da9e7accda4c68480b861fe2caea"
+                alt="picture"
+              />
+              <div class="px-0 pt-2">
+                <h5 class="card-title">
+                  <a href="#" class="font-weight-bold small text-body"
+                    >Will the Protesters Turn on Washington and Jefferson?</a
+                  >
+                </h5>
+              </div>
+            </div>
+            <div class="border-bottom pb-2">
+              <img
+                class="pt-2 mr-3 float-left img-fluid"
+                src="https://d.newsweek.com/en/full/1598987/george-washington.webp?w=95&h=95&f=6250da9e7accda4c68480b861fe2caea"
+                alt="picture"
+              />
+              <div class="px-0 pt-2">
+                <h5 class="card-title">
+                  <a href="#" class="font-weight-bold small text-body"
+                    >Will the Protesters Turn on Washington and Jefferson?</a
+                  >
+                </h5>
+              </div>
+            </div>
+            <div class="border-bottom pb-2">
+              <img
+                class="pt-2 mr-3 float-left img-fluid"
+                src="https://d.newsweek.com/en/full/1598987/george-washington.webp?w=95&h=95&f=6250da9e7accda4c68480b861fe2caea"
+                alt="picture"
+              />
+              <div class="px-0 pt-2">
+                <h5 class="card-title">
+                  <a href="#" class="font-weight-bold small text-body"
+                    >Will the Protesters Turn on Washington and Jefferson?</a
+                  >
+                </h5>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <!-- subscribe and newsletter -->
+      <div class="container-fluid row table">
+        <div class="col-lg-8 d-flex flex-column">
```

Comment:
The magazine image is leaking out of its container.

| Source Page	| Clone Page |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/57726348/87222137-f3b94880-c38e-11ea-9449-acf927ea021f.png)|![Annotation 2020-07-11 155039](https://user-images.githubusercontent.com/57726348/87222127-de441e80-c38e-11ea-85ea-60c7f713a3a6.jpg)|

Reply:
Done.